data/one_player_sleep/functions/uninstall.mcfunction

```diff
@@ -1,13 +1,16 @@
 ##########################################
 # Author: MadCat (youtube.com/MadCatHoG) 
 # Date: Jun 9, 2017
-# Version: 1.1
-# Minecraft Version 1.12
+# Version: 2.0
+# Minecraft Version 1.13
 # Description:
 # Uninstalls One Player Sleep Function
 ##########################################
 
-gamerule gameLoopFunction One-Player-Sleep:OPS_loop false
 scoreboard objectives remove OPSkickBed
 scoreboard objectives remove OPSsleep
-tellraw @a ["",{"text":"One Player Sleep v1.1","bold":true,"color":"red"},{"text":" by","color":"gold"},{"text":" "},{"text":"MadCat","bold":true,"color":"green"},{"text":"\n"},{"text":"Successfully Uninstalled!","color":"green"},{"text":"\n"},{"text":"Visit","color":"gold"},{"text":" "},{"text":"www.youtube.com/MadCatHoG","bold":true,"color":"green","clickEvent":{"action":"open_url","value":"http://www.youtube.com/MadCatHoG"}}]
\ No newline at end of file
+
+datapack disable file/one_player_sleep.zip
```

Comment:
This originally felt the the closest to what was originally there, but I perhaps it would be better to omit this and have people execute it manually if they wish to disable